account_fiscal_position_partner_type/hooks.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 FactorLibre - Rodrigo Bonilla <rodrigo.bonilla@factorlibre.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+def pre_init_hook(cr):
+    cr.execute(
+        """
+ALTER TABLE res_company
+ADD COLUMN default_fiscal_position_type character varying DEFAULT ''
```

Comment:
This is incorrect, as it's putting a default for ALL new values, and the default is not correct (it should be null, not empty string). Anyways, this is not needed at all. See below.